### PR TITLE
Hotfix#2714 part2

### PR DESCRIPTION
**Pull request by ioaNikas**
Bug fixes, text changes, more secure inputs (no negative values)...

### CODE REVIEW

File: apps/catalog/catalog/src/app/marketplace/title/avails/avails.component.ts [hunk 1]
```diff
@@ -137,4 +137,12 @@ export class MarketplaceMovieAvailsComponent {
     this.hoveredTerritory = null;
   }
 
-}
\ No newline at end of file
+  get incompleteForm() {
```
Comment (GrandSchtroumpf) [inline apps/catalog/catalog/src/app/marketplace/title/avails/avails.component.ts:140]: Would be better to have a validator for that & check if your form is valid instead.

File: apps/catalog/catalog/src/app/marketplace/title/main/main.component.scss [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 mat-card {
   margin-top: 20px;
+  width: 100%;
+  min-width: 300px;
```
Comment (GrandSchtroumpf) [inline apps/catalog/catalog/src/app/marketplace/title/main/main.component.scss:6]: I'm not sure what style you want to apply to this card ... If you're trying to do responsive, then it's better to be explicit with media queries.

File: libs/contract/src/lib/version/form/payment-schedule/payment-schedule.component.html [hunk 3]
```diff
@@ -110,6 +119,9 @@ <h3>PAYMENT TERM</h3>
         <mat-form-field appearance="outline" formGroupName="floatingDuration">
           <mat-label>Duration</mat-label>
           <input test-id="duration" matInput type="number" formControlName="duration">
+          <mat-error *ngIf="control.get('date').get('floatingDuration').get('duration').hasError('min')">
```
Comment (GrandSchtroumpf) [inline libs/contract/src/lib/version/form/payment-schedule/payment-schedule.component.html:122]: I think (but I'm not sure) that "get" can take a full path : `get('date.floatingDuration.duration')`

File: libs/contract/src/lib/version/form/payment-schedule/payment-schedule.form.ts [hunk 4]
```diff
@@ -1,11 +1,11 @@
-import { FormControl } from '@angular/forms';
+import { FormControl, Validators } from '@angular/forms';
 import { DistributionRightFloatingDurationForm } from '@blockframes/distribution-rights/form/terms/terms.form';
 import { PaymentSchedule, ScheduledDateRaw } from '@blockframes/utils/common-interfaces';
 import { FormEntity } from '@blockframes/utils/form/forms/entity.form';
 
 function createContractVersionPaymentScheduleControls(paymentSchedule: Partial<PaymentSchedule>) {
     return {
-        percentage: new FormControl(paymentSchedule.percentage),
+        percentage: new FormControl(paymentSchedule.percentage, Validators.min(1)),
```
Comment (GrandSchtroumpf) [inline libs/contract/src/lib/version/form/payment-schedule/payment-schedule.form.ts:8]: better go to min(0)

File: libs/movie/src/lib/distribution-rights/form/terms/terms.form.ts [hunk 5]
```diff
@@ -24,7 +24,7 @@ export class DistributionRightTermsForm extends FormEntity<DistributionRightTerm
 function createFloatingDuration(floating: Partial<FloatingDuration>) {
   return {
     unit: new FormControl(floating.unit),
-    duration: new FormControl(floating.duration),
+    duration: new FormControl(floating.duration, Validators.min(1)),
```
Comment (GrandSchtroumpf) [inline libs/movie/src/lib/distribution-rights/form/terms/terms.form.ts:27]: same here

File: libs/movie/src/lib/movie/form/budget/box-office/box-office.component.ts [hunk 6]
```diff
@@ -8,7 +8,7 @@ function toUnit(unit: UnitBox) {
   switch (unit) {
     case 'boxoffice_dollar': return '$'
     case 'boxoffice_euro': return '€'
-    case 'entrances': return 'entrances'
+    case 'admissions': return 'admissions'
```
Comment (GrandSchtroumpf) [inline libs/movie/src/lib/movie/form/budget/box-office/box-office.component.ts:11]: does it impact the database ? If yes do we have a migration script ?

File: libs/ui/src/lib/missing-control/missing-control.component.scss [hunk 7]
```diff
@@ -1,3 +1,7 @@
+:host {
+  margin-right: 5px;
```
Comment (GrandSchtroumpf) [inline libs/ui/src/lib/missing-control/missing-control.component.scss:2]: Shouldn't it be 8px ? Mathild told me once every margin are a multiple of 8

File: apps/catalog/catalog/src/app/marketplace/title/main/main.component.scss (hunk 2 above)
Comment (ioaNikas) [reply apps/catalog/catalog/src/app/marketplace/title/main/main.component.scss:6]: I just moved some code from a duplicate `mat-card { }` below. It didn't change the design :)

File: libs/movie/src/lib/movie/form/budget/box-office/box-office.component.ts (hunk 6 above)
Comment (ioaNikas) [reply libs/movie/src/lib/movie/form/budget/box-office/box-office.component.ts:11]: Just talked with @Jackseed , checked if films from prod already had this field, but no one got it so no migration is needed.

**Pull request merged**